addons/website_event/views/event_templates.xml

```diff
@@ -11,710 +11,796 @@
+                <nav class="navbar navbar-light border-top shadow-sm d-print-none">
+                    <div class="container">
+                        <a href="/event" class="navbar-brand h4 my-0 mr-4">
+                            <i class="fa fa-long-arrow-left text-primary mr-2"/>All Events
+                        </a>
+                        <ul class="navbar-nav flex-row">
+                            <li class="nav-item mr-3">
+                                <a t-attf-href="/event?type=#{event.event_type_id.id}" t-if="event.event_type_id" class="nav-link">
+                                    <i class="fa fa-folder-open text-primary mr-2"/><t t-esc="event.event_type_id.name"/>
+                                </a>
+                            </li>
+                            <li class="nav-item mr-3">
+                                <a t-if="event.country_id" t-attf-href="/event?country=#{event.country_id.id}" class="nav-link">
+                                    <i class="fa fa-map-marker text-primary mr-2"/><t t-esc="event.country_id.name"/>
+                                </a>
+                            </li>
+                        </ul>
+                        <t t-call="website_event.search_box">
+                            <t t-set="_classes" t-value="'form-inline ml-auto'"/>
+                        </t>
+                    </div>
+                </nav>
+            </t>
+            <t t-else="">
+                <nav class="navbar navbar-light border-top shadow-sm navbar-expand-md">
+                    <div class="container">
+                        <a href="#" t-field="event.name" class="navbar-brand h4 my-0 mr-4"/>
+                        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#o_we_event_submenu" aria-controls="o_we_event_submenu" aria-expanded="false" aria-label="Toggle navigation">
+                            <span class="navbar-toggler-icon"></span>
+                        </button>
+                        <div id="o_we_event_submenu" class="collapse navbar-collapse">
+                            <ul class="navbar-nav w-100">
+                                <t t-foreach="event.menu_id.child_id" t-as="submenu">
+                                    <t t-call="website.submenu">
+                                        <t t-set="item_class" t-value="'nav-item'"/>
+                                        <t t-set="link_class" t-value="'nav-link'"/>
+                                    </t>
+                                </t>
+                            </ul>
+                        </div>
+                    </div>
+                </nav>
+            </t>
+            <!-- Content -->
+            <t t-raw="0"/>
+            <!-- TODO - Edition message + Mess up with event multi-pages option -->
+            <div class="alert alert-info alert-dismissable mt16 css_non_editable_mode_hidden o_not_editable" t-ignore="True" role="status">
```

Comment:
That kind of stuff can be handle with data-editor-sub-message like I explained you above ;)